tests/unit_tests/test_failure_detector.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import os
+from conf import config
+
+
+from unittest.mock import Mock, MagicMock
+from resolve.resolver import Resolver
+from resolve.enums import Function, Module
+from modules.replication.models.client_request import ClientRequest
+from modules.replication.models.operation import Operation
+from modules.enums import OperationEnums
+from modules.primary_monitoring.failure_detector import FailureDetectorModule
+
+class TestConfig(unittest.TestCase):
+
+    def setUp(self):
+        self.resolver = Resolver()
+        self.clientRequest1 = ClientRequest(1, 1, Operation(OperationEnums.APPEND, 1))
+        self.clientRequest2 = ClientRequest(2, 2, Operation(OperationEnums.APPEND, 1))
```

Comment:
Nit: Might be nice to have `clientRequest2` be `APPEND 2` instead to be able to tell them apart

Reply:
(Y)